packages/main/src/popup-utils/PopoverRegistry.ts

```diff
@@ -100,7 +99,7 @@ const clickHandler = (event: MouseEvent) => {
 			return;
 		}
 
-		if (isClickInRect(event, popup.getBoundingClientRect())) {
+		if ((popup as Popover).isClicked(event)) {
```

Comment:
it could be cleaner to have method isClicked in the Popup itself then override this method inside the Popover

this way, when someone adds a new type of Popup, this code will still work